platform/features/conductor-redux/src/README.md

```diff
@@ -0,0 +1,187 @@
+Examples assume global namespace and availability from window object. 
+For now API implemented as standard standard Require JS AMDs. Could attach 
+to window from bundle.js. Perhaps attaching to window not best approach though...
+* Have not included validation (eg. start time < end time) or any other 
+business logic such as what happens when outer interval gets dragged 
+within range of inner interval. Focus is on teasing out the public API 
+right now. 
+* Time systems are vague right now also, I don't know how they're going 
+to work or whether any API has yet been specified.
+* Not clear on the differences between real-time and follow-time as it 
+concerns the time conductor? For now the API has an end bounds mode
+of FOLLOW which automatically tracks current time, and a start time mode 
+of RELATIVE. I can envision a real-time plot that is not in follow time mode, 
+but not sure what implication is for time conductor itself and how it 
+differs from an historical plot?
+* Should the time conductor be responsible for choosing time system / domain? Currently 
+it is.
+
+## Use Cases
+1. Historical session is loaded and needs to set bounds on time conductor
+2. User changes time of interest
+3. Plot controller listens for change to TOI
+4. Real-time object is loaded and time conductor is set up in real-time mode showing last 15 minutes of data.
+5. Switch to follow time. Set inner end to now, and outer end to now + 1 hour
+6. Plot controller updated on tick
+7. Plot Controller updated when user changes bounds
+8. Conductor controller needs to update bounds and mode on TC when user changes bounds
+
+### Additional possible use-cases
+1. Telemetry adapter wants to indicate presence of data at a particular time
+2. Time conductor controller wants to paint map of data availability.
+
+These use-cases could be features of the TimeConductor, but perhaps makes 
+sense to make knowledge of data availability the sole preserve of telemetry 
+adapters, not TimeConductor itself. Adapters will be ultimately responsible 
+for providing these data so doesn't make much sense to duplicate elsewhere.
+The TimeConductorController - which knows tick interval on scale (which 
+TimeConductor API does not) - could simply request data availability from 
+telemetry API and paint it into the Time Conductor UI
+
+## Example implementations of use cases
+### 1. Historical session is loaded and system sets time bounds on conductor
+``` javascript
+function loadSession(session) {
+    var tc = MCT.conductor;
+    tc.timeSystem(session.metadata.timeSystem);
+    
+    //Set start and end modes to fixed date
+    tc.startMode(tc.Modes.FIXED);
+    tc.endMode(tc.Modes.FIXED);
+    
+    //Set both inner and outer bounds
+    tc.bounds.inner.start.time(session.metadata.firstTime);
+    tc.bounds.inner.end.time(session.metadata.lastTime);
+    tc.bounds.outer.start.time(session.metadata.firstTime);
+    tc.bounds.outer.end.time(session.metadata.lastTime);
```

Comment:
Couple of things here:

* I don't believe that "outer" bounds need to be expressed as part of the time conductor at this level (the model). It's an important part of the visual control, but the information that matters to the system is the inner bounds. (Credit to @larkin for making that point to me when I was initially implementing the TC.)
* I'd also like to reduce interface depth here; this reads reasonably as an example, but starting at `tc` and getting to `tc.bounds.inner.start.time` using API docs would be a bit of an exercise.

To address both those points, I'd suggest just having a `tc.start(timestamp)` and `tc.end(timestamp)`